### PR TITLE
Some nits noted during AD review

### PR DESCRIPTION
**Pull request by kaduk**
"IV" is usually capitalized, and a couple of grammar nits.

I couldn't figure out what "PRK" meant in the (generated) output; should that
be expanded?

### CODE REVIEW

File: processlog.py [hunk 1]
```diff
@@ -252,6 +252,7 @@ def match(self, role, values):
         return None
 
 class HandleHkdf:
+    # What does 'PRK' stand for?
```
Comment (martinthomson) [inline processlog.py:255]: A question for me?  https://tools.ietf.org/html/rfc5869#section-2.2

Or did you want expand-on-first-use for this?  That's tricky, but we can do it if you think it's necessary.

File: processlog.py [hunk 2]
```diff
@@ -485,12 +486,12 @@ def report(self):
...
         log()
         if self.key_values[0][1].value != self.iv_values[0][1].value:
-            warning('key and iv have different PRK')
+            warning('key and IV have different PRK')
         self.key_values[0][1].report('PRK')
         for (n, v) in self.key_values[1:]:
             v.report('key ' + n)
         for (n, v) in self.iv_values[1:]:
-            v.report('iv ' + n)
+            v.report('IV ' + n)
```
Comment (martinthomson) [inline processlog.py:494]: I left this one lowercase, because "iv" is the label, but this is fine.

**Review by martinthomson — COMMENTED** [review]
Let me know if/when you want a new drop.  I'll need to rebuild this when the 1.3 RFC is published; the current version uses draft-28 still.

File: processlog.py (hunk 1 above)
Comment (martinthomson) [reply processlog.py:255]: f055c21 is my attempt at fixing that.  "IKM" is the other non-obvious label.

**Comment by kaduk** [comment]
I think Sean had tried to clue me in a while ago, but I only just now realized, that it would be nice if this one could be in the same cluster as the core spec, but doing that (basically) requires going through IETF LC/etc. while the test vectors are still for a draft version.  So I'd like to kick things off soon and we can see how long it takes.

Please do drop a new rev, though; no need to have genart pester us again.

**Pull request merged**